java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -13,6 +13,7 @@ public class TransactionTestUtilities {
     private static final String key3 = "{key}" + UUID.randomUUID();
     private static final String key4 = "{key}" + UUID.randomUUID();
     private static final String key5 = "{key}" + UUID.randomUUID();
+    private static final String key6 = "{key}" + UUID.randomUUID();
```

Comment:
I'm not a huge fan of this numbered key convention we've developed. It's annoying to reconcile merge conflicts when the key numbers overlap when two different people add new tests here.

Reply:
Following what they have on Py and Node tests.